### PR TITLE
[Enterprise-4.12] OBSDOCS-845 - Logging API Reference Updates 

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.12

Issue: [OBSDOCS-845](https://issues.redhat.com//browse/OBSDOCS-845)

Link to docs preview: https://74922--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/api_reference/logging-5-6-reference
https://74922--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/api_reference/logging-5-7-reference.html
https://74922--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/api_reference/logging-5-8-reference.html

QE review:
- [x] QE has approved this change.

Additional information:
Logging Version | Supported OCP Versions Matrix
5.9 | 4.15, 4.14, 4.13
5.8 | 4.12, 4.13, 4.14, 4.15
5.7 | 4.12, 4.13, 4.14
5.6 | 4.12, 4.13

PRs by branch: 
https://github.com/openshift/openshift-docs/pull/74925
https://github.com/openshift/openshift-docs/pull/74924
https://github.com/openshift/openshift-docs/pull/74923
https://github.com/openshift/openshift-docs/pull/74922

### CODE REVIEW

**Comment by libander** [comment]
@anpingli / @kabirbhartiRH - lgtm please.

**Comment by GroceryBoyJr** [comment]
https://74922--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/api_reference/logging-5-6-reference

File: observability/logging/api_reference/logging-5-7-reference.adoc [hunk 1]
```diff
@@ -1,7 +1,1310 @@
...
+This API key has been deprecated and is planned for removal in a future release. For more information, see the release notes for logging on Red{nbsp}Hat OpenShift.
+====
+
+Type:: object
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|nodeSelector|object| **(DEPRECATED)** Define which Nodes the Pods are scheduled on.
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-7-reference.adoc:970]:
```suggestion
|nodeSelector|object| **(DEPRECATED)** Define the Nodes on which the Pods are scheduled.
```

File: observability/logging/api_reference/logging-5-7-reference.adoc [hunk 2]
```diff
@@ -1,7 +1,1310 @@
...
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-7-reference.adoc:1057]: Truncated sentence

File: observability/logging/api_reference/logging-5-7-reference.adoc [hunk 3]
```diff
@@ -1,7 +1,1310 @@
...
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-7-reference.adoc:1092]: Truncated sentence

File: observability/logging/api_reference/logging-5-7-reference.adoc [hunk 4]
```diff
@@ -1,7 +1,1310 @@
...
+|value|string|  *(optional)* Value is the taint value the toleration matches to.
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-7-reference.adoc:1093]:
```suggestion
|value|string|  *(optional)* Value is the taint value the toleration matches.
```

File: observability/logging/api_reference/logging-5-7-reference.adoc [hunk 5]
```diff
@@ -1,7 +1,1310 @@
...
+|curation|object|  *(optional)*
+|logStore|object|  *(optional)*
+|visualization|object|  *(optional)*
+|======================
+
+=== .status.collection
+
+[IMPORTANT]
+====
+This API key has been deprecated and is planned for removal in a future release. For more information, see the release notes for logging on Red{nbsp}Hat OpenShift.
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-7-reference.adoc:1121]: I called this out previously - I leave it to you to determine if the term "{OCP}" even CAN be substituted - could be a problem with the curly braces?

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 6]
```diff
@@ -1,7 +1,1569 @@
...
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:565]: Truncated sentence

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 7]
```diff
@@ -1,7 +1,1569 @@
...
+|value|string|  *(optional)* Value is the taint value the toleration matches to.
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:566]:
```suggestion
|value|string|  *(optional)* Value is the taint value the toleration matches.
```

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 8]
```diff
@@ -1,7 +1,1569 @@
...
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:653]: Truncated sentence

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 9]
```diff
@@ -1,7 +1,1569 @@
...
+|value|string|  *(optional)* Value is the taint value the toleration matches to.
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:654]:
```suggestion
|value|string|  *(optional)* Value is the taint value the toleration matches.
```

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 10]
```diff
@@ -1,7 +1,1569 @@
...
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:952]: Truncated sentence

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 11]
```diff
@@ -1,7 +1,1569 @@
...
+|value|string|  *(optional)* Value is the taint value the toleration matches to.
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:953]:
```suggestion
|value|string|  *(optional)* Value is the taint value the toleration matches.
```

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 12]
```diff
@@ -1,7 +1,1569 @@
...
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:1212]:
```suggestion
|value|string|  *(optional)* Value is the taint value the toleration matches.
```

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 13]
```diff
@@ -1,7 +1,1569 @@
...
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:1211]: Truncated sentence

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 14]
```diff
@@ -1,7 +1,1569 @@
...
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:1246]: Truncated sentence

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 15]
```diff
@@ -1,7 +1,1569 @@
...
+|value|string|  *(optional)* Value is the taint value the toleration matches to.
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:1247]:
```suggestion
|value|string|  *(optional)* Value is the taint value the toleration matches.
```

File: observability/logging/api_reference/logging-5-8-reference.adoc [hunk 16]
```diff
@@ -1,7 +1,1569 @@
...
```
Comment (GroceryBoyJr) [inline observability/logging/api_reference/logging-5-8-reference.adoc:1547]:
```suggestion
|value|string|  *(optional)* Value is the taint value the toleration matches.
```

**Review by GroceryBoyJr — COMMENTED** [review]
@libander peer review done, the same issues occurred repeatedly in each of the 5.6, 5.7 and 5.8 logging reference .adoc files. I called out everything I could find in the 5.6 file, then used what I found there to search in the 5.7 and 5.8 files. If the remaining PR's feature the same issues, I will summarize rather than going through the entire PR and repeating what I've already done.

**Comment by GroceryBoyJr** [comment]
Also, be aware, @libander you need to re-create the three previews for this one since the section was moved under "observability".

**Pull request closed without merging**